portal/common/package.json

```diff
@@ -2,6 +2,7 @@
 		"dependencies": {
 				"@mysten/bcs": "^1.0.2",
 				"@mysten/sui": "^1.1.2",
+				"@noble/hashes": "^1.5.0",
```

Comment:
Question: could we use the [web cryptography api](https://www.w3.org/TR/WebCryptoAPI/#dfn-SubtleCrypto-method-digest) standard instead?

Reply:
Correct, we could do that, found an example on MDN docs too: https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest#supported_algorithms 

I will replace it.